### PR TITLE
Use bumpversion. 

### PR DESCRIPTION
**Pull request by bac**
Also use the wheels/generic directory for download-cache.

### CODE REVIEW

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3384/
Test FAILed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3386/
Test FAILed.

**Comment by bac** [comment]
Ran tests manually: http://ci.jujugui.org:8080/job/juju-gui/3388/

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "author": "Juju Developers",
   "name": "juju-gui",
   "description": "Juju GUI build dependencies",
-  "version": "0.1.0",
+  "version": "1.8.999",
```
Comment (jaycee) [inline package.json]: Is this actually our version now?

**Comment by jaycee** [comment]
Code looks good, though I'm confused by the version. Is that intentional or did you mean to revert it after trying out bumpversion?

:+1: assuming that's answered or reverted.

**Comment by kadams54** [comment]
LGTM :+1:

**Comment by bac** [comment]
@jaycee I meant to put that in the PR, but Uros and I decided on 1.9.x as the series for pyramid-fork.  So here I set to an absurd 1.8.x series with the thinking the next real release will be `VPART=minor make dist` and we'll bump to 1.9.0.

**Comment by bac** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci.jujugui.org:8080/job/juju-gui-merge/1155

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**